packages/cli/src/daemon/error-handler.ts

```diff
@@ -0,0 +1,20 @@
+import { ErrorRequestHandler, NextFunction, Request, Response } from 'express';
+import { DiagnosticsLogger } from '@ceramicnetwork/common';
+
+/**
+ * Generic error handling middleware for the daemon.
+ */
+export function errorHandler(logger: DiagnosticsLogger): ErrorRequestHandler {
+  return (err: Error, req: Request, res: Response, next: NextFunction) => {
+    (req as any).error = err // Allow other middlewares access the error
```

Comment:
Not sure this is accurate. It seems like the error should be on the response instead.

Reply:
Semantically, yes, better put it with response.